packages/Toast.vue

```diff
@@ -334,54 +342,61 @@ const onPointerMove = (event: PointerEvent) => {
 
   if (isAllowedToSwipe) {
     toastRef.value?.style.setProperty('--swipe-amount', `${yPosition}px`)
-  } else if (Math.abs(xPosition) > swipeStartThreshold) {
+  }
+  else if (Math.abs(xPosition) > swipeStartThreshold) {
     // User is swiping in wrong direction so we disable swipe gesture
     // for the current pointer down interaction
     pointerStartRef.value = null
   }
 }
 
 watchEffect(() => {
-  offset.value = heightIndex.value * GAP + toastsHeightBefore.value
+  offset.value = heightIndex.value * props!.gap! + toastsHeightBefore.value
```

Comment:
should I use `props?.gap!`?

gap already has a default value and the type is `number` but here I had TypeScript error which I resolved like this

`props!.gap!`


I leave the suggestion here feel free to change it

```suggestion
  offset.value = heightIndex.value * props?.gap! + toastsHeightBefore.value
```